### DISCUSSION TITLE
2211114-6

### DISCUSSION THREAD

**Discussion by toma-ko** [NAIST-SE/AttractingContributors#548] [General]
Homework(6)

This is my repository: https://github.com/toma-ko/Social_Developer/blob/master/README.md#homework6---which-do-you-like-better-rice-or-bread

Thank you for your cooperation.

**Comment by ani-hovhannisyan** [comment]
I contributed # 1, please contribute to my project back too:
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
